index.bs

```diff
@@ -256,6 +257,72 @@ A { <code class="vocab">st:hasShapeTreeDecoratorIndex</code> <code>Si</code> }
+    st:hasShapeTreeInstanceRoot &lt;/data/CommonNotes/&gt; ;
+    st:node &lt;/data/CommonNotes/Note1#node2&gt; ;
+    st:shape &lt;http://commonnote.example/schema#NotesContainer&gt; ;
+  ], [
+    st:hasRootShapeTree &lt;http://mysite.example/shapeTrees/management#UserData&gt; ;
+    st:hasShapeTree &lt;http://mysite.example/shapeTrees/management#Container&gt; ;
+    st:hasShapeTreeInstanceRoot &lt;/data/&gt; ;
+    st:node &lt;/data/CommonNotes/Note1#node3&gt; ; # Not necessarily the same node.
+    st:shape &lt;http://mysite.example/shapes#MyContainerShape&gt; ;
+  ]
+.
+    		</pre>
+                [[try it](http://shex.io/webapps/shex.js/doc/shex-simple?schema=PREFIX%20st%3A%20%3Chttp%3A%2F%2Fwww.w3.org%2Fns%2Fst%23%3E%0APREFIX%20xsd%3A%20%3Chttp%3A%2F%2Fwww.w3.org%2F2001%2FXMLSchema%23%3E%0A%0A%3C%23ShapeTreeLocatorShape%3E%20%7B%0A%20%20a%20%5Bst%3AShapeTreeLocator%5D%20%3B%0A%20%20st%3Alocation%20%7B%0A%20%20%20%20st%3AhasRootShapeTree%20IRI%20%3B%0A%20%20%20%20st%3AhasShapeTree%20IRI%20%3B%0A%20%20%20%20st%3AhasShapeTreeInstanceRoot%20IRI%20%3B%0A%20%20%20%20st%3Anode%20IRI%20%3B%0A%20%20%20%20st%3Ashape%20IRI%20%3B%0A%20%20%7D%2B%0A%7D&amp;data=PREFIX%20st%3A%20%3Chttp%3A%2F%2Fwww.w3.org%2Fns%2Fst%23%3E%0A%0A%3C%23loc%3E%0A%20%20a%20st%3AShapeTreeLocator%20%3B%0A%20%20st%3Alocation%20%5B%0A%20%20%20%20st%3AhasRootShapeTree%20%3Chttp%3A%2F%2Fcommonnote.example%2Fcommonnote%23NotesContainer%3E%20%3B%0A%20%20%20%20st%3AhasShapeTree%20%3Chttp%3A%2F%2Fcommonnote.example%2Fcommonnote%23Note%3E%20%3B%0A%20%20%20%20st%3AhasShapeTreeInstanceRoot%20%3C%2Fdata%2FCommonNotes%2F%3E%20%3B%0A%20%20%20%20st%3Anode%20%3C%2Fdata%2FCommonNotes%2FNote1%23node2%3E%20%3B%0A%20%20%20%20st%3Ashape%20%3Chttp%3A%2F%2Fcommonnote.example%2Fschema%23NotesContainer%3E%20%3B%0A%20%20%5D%2C%20%5B%0A%20%20%20%20st%3AhasRootShapeTree%20%3Chttp%3A%2F%2Fmysite.example%2FshapeTrees%2Fmanagement%23UserData%3E%20%3B%0A%20%20%20%20st%3AhasShapeTree%20%3Chttp%3A%2F%2Fmysite.example%2FshapeTrees%2Fmanagement%23Container%3E%20%3B%0A%20%20%20%20st%3AhasShapeTreeInstanceRoot%20%3C%2Fdata%2F%3E%20%3B%0A%20%20%20%20st%3Anode%20%3C%2Fdata%2FCommonNotes%2FNote1%23node3%3E%20%3B%20%23%20Not%20necessarily%20the%20same%20node.%0A%20%20%20%20st%3Ashape%20%3Chttp%3A%2F%2Fmysite.example%2Fshapes%23MyContainerShape%3E%20%3B%0A%20%20%5D%0A.%0A&amp;shape-map=%3C%23loc%3E%40%3C%23ShapeTreeLocatorShape%3E&amp;interface=human&amp;success=proof&amp;regexpEngine=eval-threaded-nerr&amp;manifestURL=)]
+    	</figure>
+
+These assert that, in the described document:
+<ul>
+  <li>(described by the upper <code class="vocab">st:location</code>)<ul>
+    <li>The RDF node <code>&lt;/data/CommonNotes/Note1#node2&gt;</code> conforms to the shape <code>&lt;http://commonnote.example/schema#Note&gt;</code></li>
+    <li>The Container <code>&lt;/data/CommonNotes/&gt;</code> is the root of this Managed Hierarchy and conforms to the shape <code>&lt;http://commonnote.example/schema#NotesContainer&gt;</code></li>
```

Comment:
Should this instead talk about conformance to the shape tree at the root? It's not clear to the reader whether that shape tree validates a shape.

```suggestion
    <li>The Container <code>&lt;/data/CommonNotes/&gt;</code> is the root of this Managed Hierarchy and conforms to the shape tree <code>&lt;http://commonnote.example/commonnote#NotesContainer&gt;</code></li>
```